### ISSUE TITLE
Pixi installs always in latest version ignoring version in npm install

### ISSUE THREAD

**Issue by krtr** [pixijs/pixi.js#6208]
### Expected Behavior
Pixi is installed in version specified 

### Current Behavior
Pixi installs current latest version with no respect to `package.json` / `npm install` with @

### Possible Solution
Remove `^` from version strings in  https://github.com/pixijs/pixi.js/blob/dev/bundles/pixi.js/package.json
### Steps to Reproduce
```
npm init
npm install pixi.js@5.1.5 --save --save-exact
```
5.2.0 will be installed
![image](https://user-images.githubusercontent.com/8374421/68390661-0deab900-0166-11ea-920f-23480b8af6e2.png)

### Environment

- **`pixi.js` version**: 5.2.0 (from my quick look all Pixi v5 versions do apply)
- **Browser & Version**: does not apply
- **OS & Version**: Ubuntu 19.10 / Windows 10
- **Running Example**: https://github.com/krtr/not-respectful-pixi
(error initially discovered in webpack project)

p.s.
In my project this blocks development, because updating to 5.2.0 causes application to have new issues not present in previous versions.

**Comment by bigtimebuddy** [comment]
You need to pin all the dependencies (anything package name “@pixi/...”) because the main package is a pretty empty shell to simply include dependencies. The other alternative is to use the dist build in 5.1.5, to get the locked version with all dependencies bundled. Both are obviously annoying to do, so when I do the next release I’ll make sure to pin exact versions when I bump.

**Comment by krtr** [comment]
@bigtimebuddy thanks for quick reply

>I’ll make sure to pin exact versions when I bump

:+1: 

>You need to pin all the dependencies 

I have already done that with yarn resolutions:
for anyone experiencing same issue:
```
"resolutions": {
    "@pixi/accessibility": "5.1.5",
    "@pixi/app": "5.1.5",
    "@pixi/constants": "5.1.0",
    "@pixi/core": "5.1.5",
    "@pixi/display": "5.1.3",
    "@pixi/extract": "5.1.5",
    "@pixi/filter-alpha": "5.1.5",
    "@pixi/filter-blur": "5.1.5",
    "@pixi/filter-color-matrix": "5.1.5",
    "@pixi/filter-displacement": "5.1.5",
    "@pixi/filter-fxaa": "5.1.5",
    "@pixi/filter-noise": "5.1.5",
    "@pixi/graphics": "5.1.5",
    "@pixi/interaction": "5.1.5",
    "@pixi/loaders": "5.1.5",
    "@pixi/math": "5.1.0",
    "@pixi/mesh": "5.1.5",
    "@pixi/mesh-extras": "5.1.5",
    "@pixi/mixin-cache-as-bitmap": "5.1.5",
    "@pixi/mixin-get-child-by-name": "5.1.3",
    "@pixi/mixin-get-global-position": "5.1.3",
    "@pixi/particles": "5.1.5",
    "@pixi/polyfill": "5.1.0",
    "@pixi/prepare": "5.1.5",
    "@pixi/runner": "5.1.1",
    "@pixi/settings": "5.1.3",
    "@pixi/sprite": "5.1.5",
    "@pixi/sprite-animated": "5.1.5",
    "@pixi/sprite-tiling": "5.1.5",
    "@pixi/spritesheet": "5.1.5",
    "@pixi/text": "5.1.5",
    "@pixi/text-bitmap": "5.1.5",
    "@pixi/ticker": "5.1.3",
    "@pixi/utils": "5.1.3"
  }
```

**Comment by adam-mvk** [comment]
@bigtimebuddy This isn't actually fixed in 5.1.6 unfortunately. I've just done yarn install pixi@5.1.6, and in the @pixi/ directory within node modules, it still points to 5.2.1

**Comment by bigtimebuddy** [comment]
@adam-mvk Below seemed to work for me. Please provide some steps to reproduce 5.2.1.

```bash
yarn add pixi.js@5.1.6
yarn add v1.17.3
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 33 new dependencies.
info Direct dependencies
└─ pixi.js@5.1.6
info All dependencies
├─ @pixi/accessibility@5.1.6
├─ @pixi/app@5.1.6
├─ @pixi/extract@5.1.6
├─ @pixi/filter-alpha@5.1.6
├─ @pixi/filter-blur@5.1.6
├─ @pixi/filter-color-matrix@5.1.6
├─ @pixi/filter-displacement@5.1.6
├─ @pixi/filter-fxaa@5.1.6
├─ @pixi/filter-noise@5.1.6
├─ @pixi/interaction@5.1.6
├─ @pixi/mesh-extras@5.1.6
├─ @pixi/mixin-cache-as-bitmap@5.1.6
├─ @pixi/mixin-get-child-by-name@5.1.6
├─ @pixi/mixin-get-global-position@5.1.6
├─ @pixi/particles@5.1.6
├─ @pixi/polyfill@5.1.6
├─ @pixi/prepare@5.1.6
├─ @pixi/sprite-animated@5.1.6
├─ @pixi/sprite-tiling@5.1.6
├─ @pixi/spritesheet@5.1.6
├─ @pixi/text-bitmap@5.1.6
├─ earcut@2.2.2
├─ es6-promise-polyfill@1.2.0
├─ eventemitter3@3.1.2
├─ ismobilejs@0.5.2
├─ mini-signals@1.2.0
├─ object-assign@4.1.1
├─ parse-uri@1.0.0
├─ pixi.js@5.1.6
├─ punycode@1.3.2
├─ querystring@0.2.0
├─ resource-loader@3.0.1
└─ url@0.11.0
```
